packages/app-bridge/src/AppBridgeBlock.ts

```diff
@@ -46,6 +46,7 @@ export type BlockCommand = CommandNameValidator<
         | 'closeAssetChooser'
         | 'closeTemplateChooser'
         | 'downloadAsset'
+        | 'downloadAssetsInBulk'
```

Comment:
```suggestion
        | 'downloadAssets'
```
`downloadAssets` if you pluralize it, it infers it is in bulk.